### PR TITLE
change in function

### PR DESCRIPTION
**Pull request by frmfisc**
small change in condition shock year / year

### CODE REVIEW

File: R/financial_functions.R [hunk 1]
```diff
@@ -80,7 +80,7 @@ calculate_net_profits_shock_declining_technologies_carbon_tax <- function(data,
   data_over_shoot_increasing <- data %>% dplyr::filter(.data$overshoot_direction == "Increasing") %>%
     dplyr::mutate(
       production_compensation = .data$late_sudden - .data$baseline,
-      carbon_tax = ifelse(year > shock_year, .data$carbon_tax, 0),
+      carbon_tax = ifelse(year < shock_year, 0, .data$carbon_tax),
```
Comment (bergalli) [inline R/financial_functions.R:83]: shouldn't it be `<=` ?

Comment (frmfisc) [reply R/financial_functions.R:83]: its the same in result - i prefer these kind of clear statements

Comment (bergalli) [reply R/financial_functions.R:83]: it's not the same, with `>` the carbon tax is applied 1 year after the shock year, while with `<` the carbon tax is applied on the shock year included. i.e. 1 year of carbon tax difference.

But then maybe it's better to apply it on the shock year included, meaning your modification corrects a bug ?

Comment (frmfisc) [reply R/financial_functions.R:83]: i know thats why i change it shockyear is when carbon tax is applied ...

**Review by ABuller123 — APPROVED** [review]

File: R/financial_functions.R (hunk 1 above)
Comment (frmfisc) [reply R/financial_functions.R:83]: like before 2030 shock year would have meant 2031 has the carbon tax, but we want it to be already applied in 2030

**Pull request merged**